doc/source/querying.rst

```diff
@@ -0,0 +1,96 @@
++================+=======================================================================+===========+===========================+=====================================================================+
+| mode           | The type of query to perform (distance cutoff or number of neighbors) | str       | 'none', 'ball', 'nearest' | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| r_max          | Maximum distance to find neighbors                                    | float     | r_max > 0                 | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| r_min          | Minimum distance to find neighbors                                    | float     | 0 <= r_min < r_max        | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| num_neighbors  | Number of Neighbors                                                   | int       | num_neighbors >= 0        | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| exclude_ii     | Whether or not to include self-neighbors                              | bool      | True/False                | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| r_guess        | Initial search distance for sequence of ball queries                  | float     | r_guess > 0               |  :class:`freud.locality.AABBQuery`                                  |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| scale          | Amount to increase r_guess when not enough neighbors are found        | float     | scale > 1                 |  :class:`freud.locality.AABBQuery`                                  |
```

Comment:
"Amount to increase" implies to me that it's additive instead of multiplicative.